src/java/map/BinaryMask.java

```diff
@@ -811,6 +849,62 @@ public BinaryMask getAreasWithinSpecifiedDistanceOfEdges(int distance) {
         return newMask;
     }
 
+    public FloatMask getDistanceFieldFromPoint(int x, int y) {
```

Comment:
this should probably be static as it doesnt use any fields.

Reply:
it gets the size and such from the mask, so I think it makes sense to keep? - open to alternate suggestions though...

Reply:
ah I missed that getSize call

Reply:
although it could just take that as a parameter but doesnt matter much